tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-chart.html

```diff
@@ -199,7 +212,68 @@
         runs: Array,  // of String
         tag: String,
 
-        /** @type {X_TYPE} */ xType: String,
+        /**
+         * @type {X_TYPE}
+         */
+        xType: {
+          type: String,
+          notify: true,
```

Comment:
Why does this notify? Its parent, `tf-scalar-chart`, binds it downward-only, which makes sense to me.